### PR TITLE
RS-466: Reorganize API messages for better documentation

### PR DESCRIPTION
**Pull request by atimin**
Closes #

### Please check if the PR fulfills these requirements

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] CHANGELOG.md has been updated (for bug fixes / features / docs)

### What kind of change does this PR introduce?

Refactoring

### What was changed?

I've moved API messages into appropriate modules. This allows to generate better API references on https://www.reduct.store.

### Related issues

https://github.com/reductstore/website/pull/111

### Does this PR introduce a breaking change?

No

### CODE REVIEW

**Review by AnthonyCvn — CHANGES_REQUESTED** [review]
It looks very clean and organized with Pydantics models in the msg folder! I've found a few details with some missing comments (I guess for the automatic doc generation it's important to have a comment for all variables).

File: reduct/bucket.py [hunk 1]
```diff
@@ -289,7 +206,7 @@ async def read(
             timestamp: UNIX timestamp in microseconds - if None: get the latest record
             head: if True: get only the header of a recod with metadata
         Returns:
-            async context with a record
+            Record: async context with a record
```
[the remark this answers is not in the document]
Comment (atimin) [reply reduct/bucket.py:209]: AsyncContextManager

File: reduct/bucket.py [hunk 2]
```diff
@@ -289,7 +207,7 @@ async def read(
             timestamp: UNIX timestamp in microseconds - if None: get the latest record
             head: if True: get only the header of a recod with metadata
         Returns:
-            async context with a record
+            AsyncContextManager[Record]: the record object
```
Comment (AnthonyCvn) [inline reduct/bucket.py:210]: Also the return type of the function itself to check (-> Record)

**Review by AnthonyCvn — APPROVED** [review]
Found a detail but then it's ready to merge.

File: reduct/bucket.py (hunk 2 above)
Comment (atimin) [reply reduct/bucket.py]: On no, you was right. It must be AsyncIterator . https://stackoverflow.com/questions/63125259/what-is-the-proper-way-to-type-hint-the-return-value-of-an-asynccontextmanager

**Pull request merged**